metashare/api/migrations/0002_githubrepository_product_productslug.py

```diff
@@ -32,12 +31,7 @@ class Migration(migrations.Migration):
                 ("created_at", models.DateTimeField(auto_now_add=True)),
                 ("edited_at", models.DateTimeField(auto_now=True)),
                 ("name", models.CharField(max_length=50, unique=True)),
-                (
-                    "repo_url",
-                    models.URLField(
-                        unique=True, validators=[metashare.api.gh.validate_gh_url]
-                    ),
-                ),
```

Comment:
Maybe this is a bad idea? I wasn't sure what to do when a validator used in an early migration gets removed.

Reply:
This is why to inline validators into that migration. But if you're starting from zero, this will be fine (no data), and if you're starting from past this, then it also won't matter.